api/src/main/java/com/codeforcommunity/dto/team/InviteUserRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class InviteUserRequest extends ApiDto {
```

Comment:
⛏️ InviteUsersRequest, for the list<users>

Reply:
what do you mean by this?

Reply:
semantically, it should be a plural "users" as this request contains a list of users

Reply:
done